src/ActionMenu.tsx

```diff
@@ -64,7 +69,7 @@ const ActionMenuBase = ({
   }, [open])
 
   return (
-    <AnchoredOverlay renderAnchor={renderMenuAnchor} open={open} onOpen={onOpen} onClose={onClose}>
+    <AnchoredOverlay renderAnchor={renderMenuAnchor} open={!!combinedOpenState} onOpen={onOpen} onClose={onClose}>
```

Comment:
Is the `!!` necessary here?  The types for `useProvidedStateOrCreate` look like they should always have `boolean` for `combinedOpenState`

Reply:
@dgreif I believe that `combinedOpenState` can also be `undefined` as a result of the generics used in the hooks. Maybe there was a better way to implement that?